tiltak/src/main/kotlin/no/nav/amt/tiltak/nav_enhet/NavEnhetServiceImpl.kt

```diff
@@ -9,32 +10,47 @@ import java.util.*
 
 @Service
 open class NavEnhetServiceImpl(
+	private val norgClient: NorgClient,
 	private val navEnhetRepository: NavEnhetRepository,
 	private val veilarbarenaClient: VeilarbarenaClient
 ) : NavEnhetService {
 
-	override fun hentNavEnheter(enhetIder: List<String>): List<NavEnhet> {
-		return navEnhetRepository.hentEnheter(enhetIder)
-			.map { NavEnhet(id = it.id, enhetId = it.enhetId, navn = it.navn) }
-	}
-
-	override fun upsertNavEnhet(enhetId: String, navn: String) {
-		navEnhetRepository.upsert(enhetId, navn)
-	}
-
 	override fun getNavEnhetForBruker(fodselsnummer: String): NavEnhet? {
-		return veilarbarenaClient.hentBrukerOppfolgingsenhetId(fodselsnummer)?.let { enhetId ->
-			return@let navEnhetRepository.hentEnhet(enhetId)
-				.also {
-					if (it == null) {
-						secureLog.warn("Bruker med fnr=$fodselsnummer har enhetId=$enhetId som ikke finnes i databasen/norg")
-					}
+		val oppfolgingsenhetId = veilarbarenaClient.hentBrukerOppfolgingsenhetId(fodselsnummer)
+			?: return null
+
+		return getNavEnhet(oppfolgingsenhetId)
+			.also {
+				if (it == null) {
+					secureLog.warn("Bruker med fnr=$fodselsnummer har enhetId=$oppfolgingsenhetId som ikke finnes i norg")
 				}
-		}?.toNavEnhet()
+			}
 	}
 
-	override fun getNavEnhet(enhetId: String) = navEnhetRepository.hentEnhet(enhetId)?.toNavEnhet()
+	override fun getNavEnhet(enhetId: String): NavEnhet? {
```

Comment:
Personlig tror jeg at jeg ville syntes at denne metoden blir mer lesbar hvis hele greie one-lines

Men tenker den som skrev koden sine preferanser bør være det som velges. You decide 😄

Reply:
Enig 👍 
Er kanskje litt for vant til java 😅